src/main/java/cz/xtf/openshift/OpenShiftUtil.java

```diff
@@ -828,4 +786,126 @@ private Path storeLog(String log, Path dirPath, String fileName) throws IOExcept
 
 		return filePath;
 	}
+
+	private OpenShiftWaiters waiters = new OpenShiftWaiters();
+
+	// Waiting
+	public OpenShiftWaiters waiters() {
+		return waiters;
+	}
+
+	public class OpenShiftWaiters {
```

Comment:
Could it be in separate file? This file is long enough by itself :)

Reply:
Class is now separate.